### PR TITLE
Adding ISSUE_TEMPLATE, PULL_REQUEST_TEMPLATE, and CONTRIBUTING files

### PR DESCRIPTION
**Pull request by ebeahan**
adding files to repo to address https://github.com/napalm-automation/napalm-fortios/issues/9

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Description of Issue/Question
```
Comment (mirceaulinic) [inline .github/ISSUE_TEMPLATE]: Can you add `###` at the beginning of the line to make it a header?

File: .github/ISSUE_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+### Did you follow the steps from https://github.com/napalm-automation/napalm#faq
+[ ] Yes
```
Comment (mirceaulinic) [inline .github/ISSUE_TEMPLATE]: Can do:

```
- [ ] Yes
- [x] No
```

to look like:
- [x] Yes
- [ ] No

When checked. That was missed for other drivers, but is not that important anyway.

**Review by mirceaulinic — CHANGES_REQUESTED** [review]
Two minor details

**Comment by ebeahan** [comment]
Done. Thanks!

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,32 @@
+###Description of Issue/Question
+
+
+### Did you follow the steps from https://github.com/napalm-automation/napalm#faq
+[ ] Yes
```
Comment (mirceaulinic) [inline .github/ISSUE_TEMPLATE]: Apologies, without the "x" and with hyphen at the beginning of the line, was supposed to sound like:

```
- [ ] Yes
- [ ] No
```

Comment (ebeahan) [reply .github/ISSUE_TEMPLATE]: Ah ok my bad didn't catch the hyphens.

**Comment by ebeahan** [comment]
Requested change made.

**Pull request merged**